frontend/src/pages/Profile/ConnectedProfile.scss

```diff
@@ -0,0 +1,72 @@
+@import "../../variable.scss";
+
+@mixin style1($color, $width, $height, $font-size, $margin-bottom) {
+  background: $color;
+  width: $width;
+  height: $height;
+  font-size: $font-size;
+  margin-bottom: $margin-bottom;
+}
+
+@mixin style2($width, $heigth, $margin-top) {
+  width: $width;
+  height: $heigth;
+  margin-top: $margin-top;
+}
+
+* {
+  font-family: $fontFamily;
+}
+
+main {
+  height: 100vh;
+
+  .headband {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    height: 25vh;
+    width: 100vw;
+    box-shadow: $boxShadow;
+    border-bottom: $borderDark;
+    position: relative;
+    z-index: 10;
+
+    .profilePict {
+      height: 20vh;
+      margin: 1rem;
+    }
+
+    .titleSize {
+      font-size: 1.5rem;
+      color: $colorLight;
+      text-shadow: $textShadow;
+    }
+  }
+
+  .profile {
+    background-color: $colorLight;
+    height: 75vh;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .style-button2 {
+    @include style1($gradient90deg, 80vw, 6vh, 1.6rem, 8vh);
+    border: 2.4px solid $colorDark;
+    border-radius: 8px 48px;
+    box-shadow: 0px 4px 4px 0px $colorDark;
+    color: $colorDark;
```

Comment:
put $colorLight instead

Reply:
No, it's another style of button. Exactly the case explained above.

Reply:
Modified passed with props